src/auth/googleauth.ts

```diff
@@ -59,6 +61,28 @@ interface CredentialResult {
   default: {email: string;};
 }
 
+export interface GoogleAuthOptions {
+  /**
+   * Path to a .json, .pem, or .p12 key file
+   */
+  keyFilename?: string;
```

Comment:
Doesn't google-auto-auth also accept `keyFile`?

Reply:
It does!  This wasn't documented and I was mostly basing things off of the readme.  Added :)